packages/server/src/api/controllers/static/index.ts

```diff
@@ -100,6 +100,10 @@ export const deleteObjects = async function (ctx: any) {
 }
 
 export const serveApp = async function (ctx: any) {
+  const bbHeaderEmbed = ctx.request.get("x-budibase-embed")
+    ? ctx.get("x-budibase-embed").toLowerCase() == "true"
```

Comment:
NAB: may as well use `===` here since we're directly comparing two strings. Could also make this a little terser with optional chaining:
```js
const bbHeaderEmbed = ctx.request.get("x-budibase-embed")?.toLowerCase() === "true"
```